### PR TITLE
Installed linters and set my portfolio project

### PR DESCRIPTION
**Pull request by sidiksam**
my portfolio project:
1. set up linters 
2. create an HTML file
3. create a CSS file 
4. build a navigation menu 
5. section 
6. social media  icons

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  margin: 0;
+  padding: 0;
+  background: #191765;
+  color: #fff;
+}
+
+.mobile-nav {
+  margin: 8px 18px;
+  height: 48px;
+  width: 380px;
```
Comment (omarsalem7) [inline css/style.css:12]: - [ ] Kindly remove this width to improve your design and make it responsive to all screens not specific width. One of the best practice don't give width for container let him take a default space.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  width: 30px;
+}
+
+.menu-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 8px 18px;
+  height: 100vh;
+  width: 380px;
```
Comment (omarsalem7) [inline css/style.css:28]: - [ ] Also remove width from here as stated before.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    <header>
+      <nav>
+        <div class="mobile-nav">
+          <p>John Doe</p>
+          <img class="image-nav" src="image/Menu.png" alt="No Image" />
+        </div>
+      </nav>
+    </header>
+    <div class="menu-section">
+      <p class="laura">Laura John</p>
```
Comment (omarsalem7) [inline index.html:20]: - [ ] The container of the section is a `section` tag
- [ ] The main title is a `h1` tag not `p`

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @sidiksam 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
The content is aligned with flexbox ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] It will be better if not give width for every element in page, I suggest to you leave it take a default width because this will help you when try to build desktop version, cheers. 🍻

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @sidiksam ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**